### PR TITLE
localhost not required for sqlite

### PR DESCRIPTION
**Pull request by saideep-g**
**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by saideep-g** [comment]
Localhost is not required for sqlite since it is file and no need to connect to server.

**Comment by msamogh** [comment]
Thanks for the PR, we'll give it a review as soon as possible.

**Comment by wochinge** [comment]
@saideep-g Thanks for this!

@erohmensing What do you think? I would actually rather go for one of these options

- doing `localhost: ""` + add in the comment that this is not needed for sqlite
- providing an example with postgresql (then the `url` makes sense)
- providing examples for both, sqlite and postgres

**Comment by erohmensing** [comment]
> providing examples for both, sqlite and postgres

@wochinge more examples are better right? 😃

**Comment by wochinge** [comment]
Sounds good, Ella!

@saideep-g Would you like to add an example for postgres as part of this pr and make the current example for sqlite only?

**Comment by saideep-g** [comment]
I haven't tried postgres. If you have details, providing details for both will be helpful. Thanks.
Also, isn't MySQL supported? I was able to make it work and put the details in forum:

https://forum.rasa.com/t/could-not-create-tables-mysqldb-exceptions-operationalerror-mysql-tracker-store/14419

Also, raised a bug for this:

https://github.com/RasaHQ/rasa/issues/4270

**Comment by wochinge** [comment]
Mysql is also supported. But the only thing which changes between mysql / postgresql is the port and the SQL dialect. So I think one example for SQLite and one example for a hosted database (like Postgres) would be enough.

The postges example would look like:


```
tracker_store:
  type: sql
  dialect: "postgresql"
  url: "localhost"
  port: 5432
  username: "username"
  password: "password"
  db: "database name"
```

File: docs/api/tracker-stores.rst [hunk 1]
```diff
@@ -45,7 +45,7 @@ SQLTrackerStore
             tracker_store:
                 type: SQL
                 dialect: "sqlite"  # the dialect used to interact with the db
-                url: "localhost"  # (optional) host of the sql db
+                #url: "localhost"  # (optional) host of the sql db
```
Comment (tmbo) [inline docs/api/tracker-stores.rst]:
```suggestion
                url: ""  # (optional) host of the sql db, e.g. "localhost"
```

**Comment by saideep-g** [comment]
Thanks

**Pull request merged**